lib/mapping/MapFeaturesH3M.h

```diff
@@ -64,6 +64,7 @@ struct MapFormatFeaturesH3M
 	bool levelSOD = false;
 	bool levelWOG = false;
 	bool levelHOTA = false;
+	bool levelHOTA3 = false;

```

Comment:
In `level` in  `levelXXX` adding some new information? I get that its like logging levels but even there you say simply DEBUG, ERR?

Reply:
"new information" here is that AB/SOD/HOTA/etc features are present in map.

'levelAB' is not 'AB map' but actually any map format that includes AB features. All map formats sans ROE have multiple levelXXX variables set